### PR TITLE
Add QR code for PLpeek PrairieSchedule app

### PR DESCRIPTION
**Pull request by trombonekenny**
Exam questions have a modal with a QR code link for proctors

### CODE REVIEW

File: pages/partials/questionScorePanel.ejs [hunk 1]
```diff
@@ -31,8 +31,9 @@
       <% } %>
     </tbody>
   </table>
-  <% if (assessment.allow_issue_reporting) { %>
+  <% if (assessment.type == 'Exam' || assessment.allow_issue_reporting) { %>
```
Comment (mwest1066) [inline pages/partials/questionScorePanel.ejs]: I think we want to trigger this with `authz_data.mode == 'Exam'`, which is true if we are in the CBTF IP range. People might be giving `Exam` type assessments outside of the CBTF, for example practice exams.

Comment (trombonekenny) [reply pages/partials/questionScorePanel.ejs]: Good point. I reworked the logic and tested it against HW and Exam questions in public and exam modes.

**Comment by trombonekenny** [comment]
Like @nwalters512 predicted, I need to figure out how to pacify the external-grader updating the question renderPanelsForSubmission() for its ajax-y stuff. 

Or maybe just make another panel over there on the right that's only shown in the CBTF and doesn't interact with grading?

**Comment by trombonekenny** [comment]
This last refactor moves the button to the right of the question title. Not having it be part of the questionScorePanel.ejs simplifies things with external graders, who remotely update that panel.

**Comment by mwest1066** [comment]
I fixed a small typo in the button CSS (`btn-seconday` -> `btn-secondary`) before merging.

**Pull request merged**